File: fibonacci/fibonacci.py
#!/usr/bin/env python3

def generate_fibonacci(n):
    fibonacci_sequence = []
    if n <= 0:
        return fibonacci_sequence
    elif n == 1:
        fibonacci_sequence.append(0)
    else:
        fibonacci_sequence.extend([0, 1])
        while len(fibonacci_sequence) < n:
            next_number = fibonacci_sequence[-1] + fibonacci_sequence[-2]
            fibonacci_sequence.append(next_number)
    return fibonacci_sequence

# Example usage:
n = int(input("Enter the number of Fibonacci numbers to generate: "))
fibonacci_sequence = generate_fibonacci(n)
print(f"Fibonacci sequence of length {n}:")
print(fibonacci_sequence)

File: fibonacci/fibonacci.js
function generateFibonacci(n) {
    let fibonacciSequence = [];
    if (n <= 0) {
        return fibonacciSequence;
    } else if (n == 1) {
        fibonacciSequence.push(0);
    } else {
        fibonacciSequence.push(0, 1);
        while (fibonacciSequence.length < n) {
            const nextNumber = fibonacciSequence[fibonacciSequence.length - 1] +
                fibonacciSequence[fibonacciSequence.length - 2];
            fibonacciSequence.push(nextNumber);
        }
    }
    return fibonacciSequence;
}

const n = parseInt(prompt("Enter the number of Fibonacci numbers to generate:"));
const fibonacciSequence = generateFibonacci(n);
console.log(`Fibonacci sequence of length ${n}:`);
console.log(fibonacciSequence);
